### DISCUSSION TITLE
Licença

### DISCUSSION THREAD

**Discussion by RickWalkerOne** [TadaSoftware/PyNFe#273] [Q&A]
Para usar essa biblioteca, preciso disponibilizar o codigo fonte para todo mundo?

**Comment by juniortada** [comment]
@RickWalkerOne o PyNFe usa a licença de software GNU GPLv3.
https://github.com/TadaSoftware/PyNFe/blob/main/LICENSE
https://www.gnu.org/licenses/quick-guide-gplv3.pt-br.html
Você tem a liberdade de usar livremente. Não precisa disponibilizar seu código integrado com o PyNFe.
Seja bem vindo para contribuir com o projeto.
